climakitae/explore/warming.py

```diff
@@ -681,7 +742,18 @@ def GCM_PostageStamps_MAIN_compute(wl_viz):
 
             all_plots.opts(toolbar="below")  # Set toolbar location
             all_plots.opts(hv.opts.Layout(merge_tools=True))  # Merge toolbar
-            warm_level_dict[warmlevel] = all_plots.cols(1)
+
+            if plot_type == "image":
```

Comment:
I have been stumped on how to determine this plot type more elegantly (some sort of attribute from the plot itself). If anyone has an idea on how to better approach this, I am all ears.